.gitignore

```diff
@@ -6,9 +6,9 @@ target*/
 
 lcov.info
 
-.vscode/settings.json
-.vscode/launch.json
-.vscode/tasks.json
+**/.vscode/settings.json
```

Comment:
(sorry, this was my misunderstanding — `foo` is the same as `**/foo` but `foo/bar` is not the same as `**/foo/bar`...